### ISSUE TITLE
Missing Base64 on fresh install using LS 1.5.3

### ISSUE THREAD

**Issue by russorat** [logstash-plugins/logstash-input-http#24]
When I use a fresh download of LS 1.5.3 downloaded from: https://www.elastic.co/downloads/logstash
run
```
bin/plugin install logstash-input-http
```

and use the config:
```
input {
  http {
    host => "0.0.0.0"
    port => 7070
    threads => 4
    user => 'something'
    password => 'something'
    additional_codecs => {"application/json" => "json_lines"}
  }
}
```

I get the following error during startup:

```
The error reported is:
  uninitialized constant LogStash::Inputs::Http::Base64
org/jruby/RubyModule.java:2733:in `const_missing'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-input-http-1.0.1/lib/logstash/inputs/http.rb:82:in `register'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/pipeline.rb:149:in `start_inputs'
org/jruby/RubyArray.java:1613:in `each'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/pipeline.rb:148:in `start_inputs'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/pipeline.rb:81:in `run'
org/jruby/ext/thread/Mutex.java:149:in `synchronize'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/pipeline.rb:81:in `run'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/agent.rb:155:in `execute'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/runner.rb:91:in `run'
org/jruby/RubyProc.java:271:in `call'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/logstash-core-1.5.3-java/lib/logstash/runner.rb:96:in `run'
org/jruby/RubyProc.java:271:in `call'
/Users/username/workspace/logstash-1.5.3/vendor/bundle/jruby/1.9/gems/stud-0.0.20/lib/stud/task.rb:12:in `initialize'
```

Removing the user and password work fine. Also building the plugin from the github repo and configuring LS 1.5.3 to use the local version works fine as well.

**Comment by suyograo** [comment]
Dup of https://github.com/logstash-plugins/logstash-input-http/issues/17. Fixed in 1.5.4

To get this fix on LS 1.5.3 you can do `bin/plugin update logstash-input-http`

**Comment by jsvd** [comment]
as stated, was fixed in 1.5.4, closing
